opensearch_dashboards.json

```diff
@@ -1,6 +1,6 @@
 {
   "id": "indexManagementDashboards",
-  "version": "2.4.0.0",
+  "version": "2.5.0",
```

Comment:
2.x still use version 2.4.1 https://github.com/opensearch-project/index-management-dashboards-plugin/blob/2.x/opensearch_dashboards.json

Reply:
saw dashboard 2.x is already in 2.5, we should upgrade to 2.5 as well.